Documentation/guides/messages/xa0111.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Xamarin.Android warning XA0111
-description: XA0111 warning code
+title: Xamarin.Android error XA0111
+description: XA0111 error code
 ms.date: 05/10/2019
 ---
 # Xamarin.Android warning XA0111
 
 ## Issue
 
-Xamarin.Android could not get the `aapt2` version via the command:
+Xamarin.Android could not get the AAPT2 version via the command:
```

Comment:
Not sure we want this change?

Reply:
This is based on earlier review: https://github.com/xamarin/xamarin-android/pull/5165#discussion_r498497108